packages/hub/src/lib/parse-safetensors-metadata.ts

```diff
@@ -14,7 +14,7 @@ export const SAFETENSORS_INDEX_FILE = "model.safetensors.index.json";
 export const RE_SAFETENSORS_FILE = /\.safetensors$/;
 export const RE_SAFETENSORS_INDEX_FILE = /\.safetensors\.index\.json$/;
 export const RE_SAFETENSORS_SHARD_FILE =
-	/^(?<prefix>(?<basePrefix>.*?)[_-])(?<shard>\d{5})-of-(?<total>\d{5})\.safetensors$/;
+	/^(?<prefix>(?<basePrefix>.*?)[_-])(?<shard>\d+)-of-(?<total>\d+)\.safetensors$/;
```

Comment:
at least enforce the same number of digits 🙏  (not sure how to do this in the regex, maybe it's an OR or the different digits for 5 and 6 digits)

Reply:
or we just don't care about the 5X and just allow regular numbers to avoid this issue again in the future?

Reply:
hmm no i think here is fine (we use this hub-side anyways)

Reply:
Done here: https://github.com/huggingface/huggingface.js/pull/1817/commits/df147f8ae1a49c0058d4c7704f1833fbebb8b5a4

Supporting both 5 OR 6 digits for shards